spec/support/matchers/be_a_user_representation.rb

```diff
@@ -1,12 +1,12 @@
-RSpec::Matchers.define :be_a_user_representation do |user|
+RSpec::Matchers.define :be_a_user_representation do
   match do |json|
-    response_attributes = user.sliced_attributes %w(
```

Comment:
`#user.sliced_attributes` method has several issues:
1. It doesn't raise an exception or something when a user has no certain attributes. Because of this `response_attributes` can contain not all of expected attributes.
2. The method doesn't work with associations.